crawler/config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "index_name": "dev_fl0_docs",
-  "start_urls": ["https://documentation-dev.fl0.io"],
-  "sitemap_urls": ["https://documentation-dev.fl0.io/sitemap.xml"],
+  "index_name": "prod_fl0_docs",
+  "start_urls": ["https://documentation.fl0.io"],
```

Comment:
Shouldn't this use the canonical https://docs.fl0.com?

Reply:
@stennie yes, will update